### PR TITLE
set up models and fixtures for users

### PR DESCRIPTION
**Pull request by kmchandler**

### CODE REVIEW

**Comment by Massacre-Saint** [comment]
checking now

**Comment by shalane-proctor** [comment]
Taking a look

File: rareapi/models/user.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class User(models.Model):
+    first_name = models.CharField(max_length=50)
+    models.CharField(max_length=50)
+    bio = models.CharField(max_length=1000)
+    profule_image_url = models.URLField(max_length=200)
```
Comment (shalane-proctor) [inline rareapi/models/user.py:7]: can you change to profile_image_url?

Comment (kmchandler) [reply rareapi/models/user.py]: fixed, good catch!

**Review by shalane-proctor — APPROVED** [review]

**Review by Massacre-Saint — APPROVED** [review]
Looks great thank you!

**Pull request merged**